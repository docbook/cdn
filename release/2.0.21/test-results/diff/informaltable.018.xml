<html xmlns:deltaxml="http://www.deltaxml.com/ns/well-formed-delta-v1" xmlns="http://www.w3.org/1999/xhtml" xmlns:dxx="http://www.deltaxml.com/ns/xml-namespaced-attribute" xmlns:dxa="http://www.deltaxml.com/ns/non-namespaced-attribute" deltaxml:deltaV2="A!=B" deltaxml:version="2.0" deltaxml:content-type="full-context">
   <head deltaxml:deltaV2="A=B">
      <link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css"/>
      <link rel="stylesheet" type="text/css" href="../../resources/css/default.css"/>
      <link href="../../resources/css/prism.css" rel="stylesheet" type="text/css"/>
      <meta charset="utf-8"/>
      <meta name="generator" content="DocBook XSL 2.0 Stylesheets Vxxx"/>
      <script type="text/javascript" src="../../resources/js/dbmodnizr.js"/>
      <title>Unit Test: informaltable.018</title>
   </head>
   <body deltaxml:deltaV2="A!=B">
      <article deltaxml:deltaV2="A!=B" id="R.1" class="article">
         <header deltaxml:deltaV2="A=B" class="article-titlepage">
            <h2>Unit Test: informaltable.018</h2>
            <div class="author">
               <h3>
                  <span class="personname">
                     <span class="firstname">Adam</span>
                     <span class="surname">Di Carlo</span>
                  </span>
               </h3>
            </div>
         </header>
         <p deltaxml:deltaV2="A">Table entry whitespace trim test. Both entries should left align
the same.
</p><div deltaxml:deltaV2="B" class="content">
            <p>Table entry whitespace trim test. Both entries should left align
the same.
</p>
            <figure id="informaltable" class="informaltable-wrapper">
               <div class="informaltable">
                  <table border="0" style="border-collapse: collapse;border-top: 1px solid ; border-bottom: 1px solid ; border-left: 1px solid ; border-right: 1px solid ; ">
                     <colgroup>
                        <col class="tcol1" width="100%"/>
                     </colgroup>
                     <tbody>
                        <tr>
                           <td style="border-bottom: 1px solid ; ">
                              <p>Entry contents one.</p>
                           </td>
                        </tr>
                        <tr>
                           <td>
                              <p>
              Entry contents two.
            </p>
                           </td>
                        </tr>
                     </tbody>
                  </table>
               </div>
            </figure>
         </div><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><figure deltaxml:deltaV2="A" id="informaltable" class="informaltable-wrapper">
            <div class="informaltable">
               <table border="0" style="border-collapse: collapse;border-top: 1px solid ; border-bottom: 1px solid ; border-left: 1px solid ; border-right: 1px solid ; ">
                  <colgroup>
                     <col class="tcol1" width="100%"/>
                  </colgroup>
                  <tbody>
                     <tr>
                        <td style="border-bottom: 1px solid ; ">
                           <p>Entry contents one.</p>
                        </td>
                     </tr>
                     <tr>
                        <td>
                           <p>
              Entry contents two.
            </p>
                        </td>
                     </tr>
                  </tbody>
               </table>
            </div>
         </figure>
      </article>
      <script deltaxml:deltaV2="A=B" src="../../resources/js/prism.js"/>
   </body>
</html>