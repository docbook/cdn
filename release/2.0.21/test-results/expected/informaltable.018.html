<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Unit Test: informaltable.018</title><meta charset="utf-8" /><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.17" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /></head><body><article id="R.1" class="article"><header class="article-titlepage"><h2>Unit Test: informaltable.018</h2><div class="author"><h3><span class="personname"><span class="firstname">Adam</span> <span class="surname">Di Carlo</span></span></h3></div></header><p>Table entry whitespace trim test. Both entries should left align
the same.
</p><figure id="informaltable" class="informaltable-wrapper"><div class="informaltable"><table border="0" style="border-collapse: collapse;border-top: 1px solid ; border-bottom: 1px solid ; border-left: 1px solid ; border-right: 1px solid ; "><colgroup><col class="tcol1" width="100%" /></colgroup><tbody><tr><td style="border-bottom: 1px solid ; ">
            <p>Entry contents one.</p>
          </td></tr><tr><td>
            <p>
              Entry contents two.
            </p>
          </td></tr></tbody></table></div></figure></article><script src="../../resources/js/prism.js"></script></body></html>